test/common/tmpdir.js

```diff
@@ -18,25 +29,27 @@ const tmpdirName = '.tmp.' +
 const tmpPath = path.join(testRoot, tmpdirName);
 
 let firstRefresh = true;
-function refresh() {
-  rmSync(tmpPath);
+function refresh(useSpawn = false) {
```

Comment:
This change should be [documented](https://github.com/nodejs/node/blob/85f9b27c1444df17fdd33b987afef349d1b46231/test/common/README.md#refresh).